From 1223e274f4304dda6ea6bf718135e5e6b351af60 Mon Sep 17 00:00:00 2001
From: Michael Scott <mike@foundries.io>
Date: Sat, 26 Mar 2022 20:44:17 -0700
Subject: [PATCH] imx8mn-var-som: raise NR_DRAM_BANKS to 2

Due to OPTEE location in the middle of DRAM, we need 1 more DRAM bank.

Signed-off-by: Michael Scott <mike@foundries.io>
---
 configs/imx8mn_var_som_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configs/imx8mn_var_som_defconfig b/configs/imx8mn_var_som_defconfig
index e5e00c89e3..616bf0b3d2 100644
--- a/configs/imx8mn_var_som_defconfig
+++ b/configs/imx8mn_var_som_defconfig
@@ -18,7 +18,7 @@ CONFIG_TARGET_IMX8MN_VAR_SOM=y
 CONFIG_SPL_MMC_SUPPORT=y
 CONFIG_SPL_SERIAL_SUPPORT=y
 CONFIG_SPL_DRIVERS_MISC_SUPPORT=y
-CONFIG_NR_DRAM_BANKS=1
+CONFIG_NR_DRAM_BANKS=2
 CONFIG_SPL=y
 CONFIG_SPL_IMX_ROMAPI_LOADADDR=0x48000000
 CONFIG_CSF_SIZE=0x2000
-- 
2.35.1

